src/main/scala/com/databricks/labs/overwatch/ApiMeta.scala

```diff
@@ -0,0 +1,98 @@
+package com.databricks.labs.overwatch
+import org.apache.log4j.{Level, Logger}
+
+/**
+ * Configuration for each API.
+ */
+trait ApiMeta {
+
+  private val logger: Logger = Logger.getLogger(this.getClass)
+  var paginationKey:String="";
+  var paginationToken:String="";
+  var dataframeColumns:String="";
+  var apiCallType:String="";
+  var microBatchSize = 50
```

Comment:
This should be controlled from ApiEnv and further externalized to the global overwatch config so that it can be easily overridden from config.